src/commands/get/build.js

```diff
@@ -13,7 +13,11 @@ class BuildsCommand extends Command {
     const { flags, args } = this.parse(BuildsCommand);
     const { env, locale } = args; // package is reserved so we don't try to destructure it
     const { pkg, version } = this.parsePackage(args.package);
-    const { host, auth } = this.mergeConfig(flags);
+    const { wrhsHost: host, auth } = this.mergeConfig(flags);
+
+    if (!host) {
+      return console.log('Missing warehouse host. Please configure `~/.wrhs` config file, or use the `--host` option.');
```

Comment:
log or error? And does this result in a failed cli call so i I do `wrhs get:build ... && foo` that foo won't run?

Reply:
Also this could probably be a bit DRY-er, seems like you already have a base class here, you could put it down there and pass in a `host`/`status-host` option.  (and if you throw rather than return, you could call it `ensureHost` or something).